src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -310,6 +315,14 @@ public void setFirestoreQueryBatchSize(int firestoreQueryBatchSize) {
         this.firestoreQueryBatchSize = firestoreQueryBatchSize;
     }
 
+    public int getDrsLookupMax() {
```

Comment:
I think it's clearer if the getter/setter matches the field name it's wrapping
```suggestion
    public int getMaxDrsLookups() {
```
Either change the methods or the field so they agree.